BlueDB/src/main/java/org/bluedb/api/BlueDbVersion.java

```diff
@@ -0,0 +1,9 @@
+package org.bluedb.api;
+
+public enum BlueDbVersion {

```

Comment:
We decided to create an object for holding version information that we can serialize to and from disk. It will have three ints and a string. Remember to register the class. We may want to create an interface called BlueDbVersion and create a concretion called BlueDbVersionV1 so that we are free to change what information is stored in the version in the future without breaking serialization.

Reply:
Oh, we also decided that existing collections with no version will be given a version of 1.0.0

Reply:
- The version is only written once in the meta data, so its probably not imperative to register the class
- While creating an interface and versioning the concretions gives us a little more flexibility, its not that big of deal.